osu.Game.Rulesets.Mania/Replays/ManiaReplayFrame.cs

```diff
@@ -21,7 +21,8 @@ public ManiaReplayFrame()
         public ManiaReplayFrame(double time, params ManiaAction[] actions)
             : base(time)
         {
-            Actions.AddRange(actions);
+            if (actions.Length > 0)
```

Comment:
Is this required?

Reply:
i don't actually think so, else it would have been failing until now.